docs/using/user-accounts.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Accounts
+authors: "Tim McMackin"
+last_update:
+  date: 30 December 2024
+---
+
+From the user perspective, a Tezos account is much like an account that you might have on a web application or computer system.
+You can use a Tezos account to do many things, including:
+
+- Store and work with tez, the native currency token of the Tezos system
+- Store and work with other [tokens](/architecture/tokens)
+- Uniquely identify yourself to online applications
+- Send transactions to Tezos, such as sending tokens to another account or calling a [smart contract](/smart-contracts)
+- Sign messages to prove that they came from your account
+
+From a technical standpoint, a Tezos account is a tool that allows a user to encrypt transactions in a way that proves that those transactions came from them.
+In this way, using a Tezos account is how you prove your identity to the Tezos system and to applications that use Tezos.
+
+This page discusses user accounts.
+Other types of Tezos accounts include [smart contracts](/smart-contracts) and [Smart Rollups](/architecture/smart-rollups).
+For technical information on user accounts, see [Accounts and addresses](https://tezos.gitlab.io/active/accounts.html) in the Octez documentation.
+
+Tezos users use wallets to manage their accounts; see [Wallets](/using/wallets).
+
+## Account addresses
+
+A Tezos account address uniquely identifies an account.
+Most Tezos user account addresses start with `tz1`, as in `tz1QCVQinE8iVj1H2fckqx6oiM85CNJSK9Sx`, but they can also start with `tz2`, `tz3`, or `tz4`.
+They consist of 26 characters and include only letters and numbers.
+
+:::note
+
+Tezos account addresses are case-sensitive.
+
+:::
+
+Technically, what users refer to as the "account address" is the hash of the public key of the account.
```

Comment:
Maybe add a note/expand saying that the tz1/tz2/tz3/tz4 variation corresponds to the underlying crypto curve/pair

Reply:
Added "depending on the cryptographic method used to create them".